Formula/squashfs.rb

```diff
@@ -15,35 +15,93 @@ class Squashfs < Formula
   depends_on "lz4"
   depends_on "lzo"
   depends_on "xz"
+  depends_on "zstd"
 
   # Patch necessary to emulate the sigtimedwait process otherwise we get build failures
   # Also clang fixes, extra endianness knowledge and a bundle of other macOS fixes.
   # Originally from https://github.com/plougher/squashfs-tools/pull/3
   patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/05ae0eb1/squashfs/squashfs-osx-bundle.diff"
-    sha256 "276763d01ec675793ddb0ae293fbe82cbf96235ade0258d767b6a225a84bc75f"
+    url "https://gist.githubusercontent.com/blake-riley/b1d62c0c6eabe8209a77fbd4456124e7/raw/0818a2309e9e74fe2c63f0d01360e24228f051dc/squashfs-osx-bundle.diff"
```

Comment:
Please use the review url.

Reply:
Hi Sean, what do you mean by the 'review url'?
Do you want me to put in a PR at formula-patches, and link back to this PR?

Reply:
Sorry, mixing up terms. You can get the diff from the pull request you made upstream and use that one here.